SpatialGDK/Extras/insights-proxy/InsightsProxy.py

```diff
@@ -0,0 +1,37 @@
+import socket 
+import sys 
+
+def main():
+    is_connected = False
+    while not is_connected:
+        try:
+            print("Attempting connection to 127.0.0.1:1981 [server worker]")
+            worker_connection = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            worker_connection.connect(("127.0.0.1", 1981))
+            worker_connection.settimeout(1.0)
+            is_connected = True
+        except socket.error:
+            pass
+            
+    print("Connecting to 127.0.0.1:1980 [Insights]")
+    proxy_connection = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    proxy_connection.connect(("127.0.0.1", 1980))
```

Comment:
minor: worth putting in to variable and using `f'Connecting to {up}:{port} [Insights]'` format strings?

Reply:
Would need to upgrade to python 3.6 to do that. Python 3.0 has Connecting to `'{}:{} [Insights]'.format(ip, port)` which achieves the same, or if sticking with 2.x c style %s format is available